docs/modules/ROOT/pages/redis/create.adoc

```diff
@@ -0,0 +1,76 @@
+= Create a Redis service
+
+AppCat allows DevOps engineers to quickly provision a redis instance.
+
+== Create a redis service
+
+include::user:ROOT:partial$login-in-terminal.adoc[]
+
+. Apply the following object on your namespace, as specified by its YAML description.
++
+.Example to create an Object Storage bucket
+[source,yaml]
+----
+apiVersion: appcat.vshn.io/v1
+kind: ExoscaleRedis
+metadata:
+  name: my-redis-example <1>
+  namespace: my-namespace <2>
+spec:
+  parameters:
+    service:
+      redisSettings:
+        maxmemory_policy: noeviction <3>
+      zone: ch-dk-2 <4>
+    size:
+      plan: hobbyist-2 <5>
+  writeConnectionSecretToRef:
+    name: redis-creds <6>
+----
+<1> Instance name: *don't just copy/paste this code, remember to choose your own name here!*
```

Comment:
Why do I have to choose another instance name? That was an issue with ObjectBucket, because the buckets need a globally unique name...

Reply:
true - I copied it over verbatim without adjusting. Removed.